Assets/Score.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class Score : MonoBehaviour
+{
+    public TMP_Text score;
```

Comment:
Since this is already a TMP_Text variable, you don't need to `GetComponent<TMP_Text>` below.  Would need it if this was just a GameObject

have not opened the scene, but is this the score display when game is done?  (that's why no update() only start() ?)

Reply:
Si. This class is only used to set the score.text